.github/workflows/frontend-skipped-checks.yml

```diff
@@ -0,0 +1,53 @@
+name: Frontend
+
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+      - "**.md"
+      - "**_test.clj"
+      - "**.cy.*.js" # .cy.spec.js, .cy.snap.js
+      - "frontend/test/__support__/e2e/**"
+      - "frontend/test/__runner__/*cypress*"
+
+jobs:
+  fe-linter-prettier:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+      - run: |
+          echo "Didn't run due to conditional filtering"
+
+  fe-linter-eslint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    steps:
+      - run: |
+          echo "Didn't run due to conditional filtering"
```

Comment:
@alxnddr there is a slight risk in not running linters for some E2E helpers, but I thought that having Husky pre-commit hooks will solve that. Do you agree or do you still think it would be nice to run linters anyway?

Reply:
I think we are safe to go, the probability of getting any issues is insignificantly tiny